src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -1509,11 +1509,39 @@ namespace cryptonote
     };
   };
 
+  struct COMMAND_RPC_GET_TRANSACTION_POOL_HASHES
+  {
+    struct request
+    {
+      BEGIN_KV_SERIALIZE_MAP()
+      END_KV_SERIALIZE_MAP()
+    };
+
+    struct response
+    {
+      std::string status;
+      std::vector<std::string> tx_hashes;
+      bool untrusted;
+
+      BEGIN_KV_SERIALIZE_MAP()
+        KV_SERIALIZE(status)
+        KV_SERIALIZE(tx_hashes)
+        KV_SERIALIZE(untrusted)
+      END_KV_SERIALIZE_MAP()
+    };
+  };
+
   struct tx_backlog_entry
   {
     uint64_t blob_size;
     uint64_t fee;
     uint64_t time_in_pool;
+
+    BEGIN_KV_SERIALIZE_MAP()
+      KV_SERIALIZE(blob_size)
+      KV_SERIALIZE(fee)
+      KV_SERIALIZE(time_in_pool)
+    END_KV_SERIALIZE_MAP()
```

Comment:
Since this fix doesn't seem to be related to the main point addressed here, shouldn't it be made as a separate commit for clarity? Also, does this mean that this daemon RPC wasn't functioning properly previously?

Reply:
Unrelated, I'd started making a non binary version of that one too and decided otherwise since it's less useful. It's working fine AFAIK.